modules/standard/Random.chpl

```diff
@@ -183,6 +343,9 @@ class RandomStream {
   }    
 }
 
+// CHPLDOC FIXME: Prints "inherited from object" at the bottom which
+// seems out of place.
+
```

Comment:
We'd like to output inherited functions and fields, but since object does not define any information, there is nothing visible after that.  Should I alter chpldoc to skip printing out "inherited from object" or from printing out any class from which it inherits nothing?